validphys2/src/validphys/tests/test_pseudodata.py

```diff
@@ -80,3 +82,27 @@ def test_read_matches_recreate():
         )
         pd.testing.assert_index_equal(read.tr_idx, recreate.tr_idx, check_order=False)
         pd.testing.assert_index_equal(read.val_idx, recreate.val_idx, check_order=False)
+
+
+def test_level0_commondata_wc():
+    from validphys.loader import Loader
+    from validphys.covmats import dataset_t0_predictions
```

Comment:
Please put this outside and add a comment to explain what this is testing.

Can't we use only the loader or only the api?

Reply:
Thank you for the comment, I agree, although I think in order to use the API for both we would probably need to have  a parse_dataset function